sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -224,34 +222,32 @@ public PermissionTemplateDto insert(DbSession session, PermissionTemplateDto per
   /**
    * Each row returns a #{@link CountByProjectAndPermissionDto}
    */
-  public void usersCountByTemplateIdAndPermission(final DbSession dbSession, List<Long> templateIds, final ResultHandler resultHandler) {
-    final Map<String, Object> parameters = new HashMap<>();
+  public void usersCountByTemplateIdAndPermission(DbSession dbSession, List<Long> templateIds, ResultHandler resultHandler) {
+    Map<String, Object> parameters = new HashMap<>(1);
 
-    executeLargeInputsWithoutOutput(templateIds, new Function<List<Long>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Long> partitionedTemplateIds) {
+    executeLargeInputsWithoutOutput(
+      templateIds,
+      partitionedTemplateIds -> {
         parameters.put("templateIds", partitionedTemplateIds);
         mapper(dbSession).usersCountByTemplateIdAndPermission(parameters, resultHandler);
         return null;
-      }
-    });
+      });
   }
 
   /**
    * Each row returns a #{@link CountByProjectAndPermissionDto}
    */
-  public void groupsCountByTemplateIdAndPermission(final DbSession dbSession, final List<Long> templateIds, final ResultHandler resultHandler) {
-    final Map<String, Object> parameters = new HashMap<>();
+  public void groupsCountByTemplateIdAndPermission(DbSession dbSession, List<Long> templateIds, ResultHandler resultHandler) {
+    Map<String, Object> parameters = new HashMap<>(2);
     parameters.put(ANYONE_GROUP_PARAMETER, DefaultGroups.ANYONE);
 
-    executeLargeInputsWithoutOutput(templateIds, new Function<List<Long>, Void>() {
-      @Override
-      public Void apply(@Nonnull List<Long> partitionedTemplateIds) {
+    executeLargeInputsWithoutOutput(
+      templateIds,
+      partitionedTemplateIds -> {
         parameters.put("templateIds", partitionedTemplateIds);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)